site/templates/meet.json.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * @var Kirby\Cms\Pages $upcoming
+ * @var Kirby\Cms\Pages $past
+ */
+$upcomingEvents = [];
+$pastEvents     = [];
+
+$since = param('since');
+
+foreach ($upcoming as $event) {
+	$upcomingEvents[] = [
+		'title'    => $event->shortTitle()->value(),
+		'date'     => $event->datetime()->format('c'),
+		'timezone' => $event->timezone()->value(),
+		'link'     => $event->link()->value(),
+		'isMeetup' => $event->isMeetup(),
+	
+	];
+}
+
+// Only meetups, no other events
+$past = $past->filterBy('city', '!=', '');
+
+// Use param since to filter meetups after given date
+if ($since = param('since')) {
```

Comment:
`Cleanup`: This variable is already set in line 9:

```suggestion
if (is_string($since) === true) {
```